src/wallet/test/fuzz/crypter.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/setup_common.h>
+#include <wallet/crypter.h>
+#include <wallet/test/util.h>
+
+namespace wallet {
+namespace {
+
+const TestingSetup* g_setup;
+void initialize_crypter()
+{
+    static const auto testing_setup = MakeNoLogFileContext<const TestingSetup>();
+    g_setup = testing_setup.get();
+}
+
+FUZZ_TARGET(crypter, .init = initialize_crypter)

```

Comment:
I don't quite understand why the CI isn't failing here. You're using the new macro from #28065 but you haven't rebased past that PR, so this shouldn't work.

These are the errors that i get locally:
```
wallet/test/fuzz/crypter.cpp:22:22: error: too many arguments provided to function-like macro invocation
FUZZ_TARGET(crypter, .init = initialize_crypter)
                     ^
./test/fuzz/fuzz.h:31:9: note: macro 'FUZZ_TARGET' defined here
#define FUZZ_TARGET(name) \
        ^
wallet/test/fuzz/crypter.cpp:22:1: error: a type specifier is required for all declarations
FUZZ_TARGET(crypter, .init = initialize_crypter)
^
wallet/test/fuzz/crypter.cpp:24:5: error: 'FuzzedDataProvider' does not refer to a value
    FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
    ^
./test/fuzz/FuzzedDataProvider.h:31:7: note: declared here
class FuzzedDataProvider {
      ^
wallet/test/fuzz/crypter.cpp:89:2: error: expected ';' after top level declarator
}
 ^
 ;
4 errors generated.
```

Reply:
> Now that we are moving to persistent workers, we can even compile all commits in a pull request.

Sounds good but would that really take one or two days? With caching this should only take marginally longer, no?

Reply:
This is out of the blue, but the reason I struggle to compile after each change is that I'm currently using a Windows 10 OS. And I can't run Wallet Fuzz tests on my machine for some unknown reason. I've attempted Fuzz tests outside of Wallet code, and they run fine. I've also experimented with other Windows machines, and this issue seems to persists across all of them.

So, I've been using an Ubuntu VM for code testing. I've gone through [Makefile.test.include](https://github.com/bitcoin/bitcoin/blob/master/src/Makefile.test.include), and everything appears to be in order. I'm uncertain about the reason of this problem here.

Reply:
Yes, I'm on it. Will try to reproduce it once again and create an issue then.